indexing-service/src/main/java/io/druid/indexing/overlord/RemoteTaskRunnerWorkItem.java

```diff
@@ -30,34 +31,39 @@
 {
   private final SettableFuture<TaskStatus> result;
   private final Worker worker;
+  private TaskLocation location;
 
   public RemoteTaskRunnerWorkItem(
       String taskId,
-      Worker worker
+      Worker worker,
```

Comment:
It's used internally by the RTR (it shouldn't be exposed in the json from the overlord endpoints)